src/payment/strategies/cybersource/cybersource-payment-strategy.ts

```diff
@@ -112,7 +119,11 @@ export default class CyberSourcePaymentStrategy implements PaymentStrategy {
             );
     }
 
-    private _getOrderData(paymentData: CreditCardInstrument): CardinalOrderData {
+    private _getBinNumber(payment: CardinalSupportedPaymentInstrument): string {
+        return isVaultedInstrument(payment) ? payment.iin : payment.ccNumber;
+    }
+
+    private _getOrderData(paymentData: CardinalSupportedPaymentInstrument): CardinalOrderData {
```

Comment:
I know u haven't changed this as part of this PR, but would you mind changing it to a single call to `getState`?

```ts
const state = this._store.getState();

const billingAddress = state.billingAddress.getBillingAddress();
const shippingAddress = state.shippingAddress.getShippingAddress();
const checkout = state.checkout.getCheckout();
const order = state.order.getOrder();
```